actions/new.action.ts

```diff
@@ -152,28 +177,62 @@ const installPackages = async (
     console.info();
     return;
   }
+
   try {
     packageManager = PackageManagerFactory.create(inputPackageManager);
-    await packageManager.install(installDirectory, inputPackageManager);
+    await packageManager.install(
+      installDirectory,
+      inputPackageManager,
+      shouldInitializePrima,
+    );
   } catch (error) {
     if (error && error.message) {
       console.error(chalk.red(error.message));
     }
   }
 };
 
+const createPrismaFiles = async (
+  dryRunMode: boolean,
+  createDirectory: string,
+  shouldInitializePrima: boolean,
+) => {
+  if (!shouldInitializePrima) {
+    return;
+  }
+  if (dryRunMode) {
+    console.info();
+    console.info(chalk.green(MESSAGES.DRY_RUN_MODE));
+    console.info();
+    return;
+  }
+  const prismaInstance = new ClassPrisma();
+  try {
+    await prismaInstance.create(createDirectory);
+  } catch (error) {
+    console.error('could not generate the prisma files successfully');
+  }
+};
+
 const askForPackageManager = async (): Promise<Answers> => {
   const questions: Question[] = [
     generateSelect('packageManager')(MESSAGES.PACKAGE_MANAGER_QUESTION)([
       PackageManager.NPM,
       PackageManager.YARN,
-      PackageManager.PNPM,
```

Comment:
why is this removed?

Reply:
That was a mistake made while testing out the changes. it has been rectified in the latest commit.